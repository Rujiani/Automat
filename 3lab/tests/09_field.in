vuông c = { 4, 7, thật };

số nguyên sum;
sum = c=>x + c=>y;

sum;
c=>x;
c=>y;
c=>busy;
