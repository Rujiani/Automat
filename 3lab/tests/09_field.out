11
4
7
thật
